api/db.py

```diff
@@ -227,6 +349,13 @@ def get_user(self, username: str) -> dict:
             }
 
     def create_song_user_link(self, song_id: int, username: str):
+        """Attach a song to an user.
```

Comment:
"a user" au lieu de "an user"

Reply:
Pro tip : GitHub a des suggestions (c'est l'icône +/- à côté des icônes de formatage) :D 

```suggestion
        """Attach a song to a user.
```